mdal/frmts/mdal_cf.cpp

```diff
@@ -401,7 +406,8 @@ std::unique_ptr< MDAL::Mesh > MDAL::DriverCF::load( const std::string &fileName,
     );
     mesh->faces = faces;
     mesh->vertices = vertices;
-
+    size_t faceVerticesMaximumCount = mDimensions.size( mDimensions.MaxVerticesInFace );
+    mesh->setFaceVerticesMaximumCount( faceVerticesMaximumCount );
```

Comment:
not sure why it is needed as it is already set few lines above in https://github.com/lutraconsulting/MDAL/blob/master/mdal/frmts/mdal_cf.cpp#L397